common/app/services/S3.scala

```diff
@@ -163,10 +163,7 @@ object S3FrontsApi extends S3 {
   def listCollectionIds: List[String] = getCollectionIds(s"$location/collection/")
   def putCollectionJson(id: String, json: String) = {
     val putLocation: String = s"$location/collection/$id/collection.json"
-    if (Switches.FaciaToolPutPrivate.isSwitchedOn) {
```

Comment:
How come this has been removed? Are you considering making these private?
We probably don't want the objects public.